vision/google/cloud/vision/_gax.py

```diff
@@ -66,7 +68,12 @@ def annotate(self, images=None, requests_pb=None):
             requests = requests_pb
 
         annotator_client = self._annotator_client
-        responses = annotator_client.batch_annotate_images(requests).responses
+        try:
+            api_result = annotator_client.batch_annotate_images(requests)
+            responses = api_result.responses
+        except RetryError as exception:
```

Comment:
If you notice how I access it on the line below, it's pretty custom to the `GaxError` class. `exception.cause.exception()`.